CoYFrontend/src/app/app.component.scss

```diff
@@ -1,21 +1,31 @@
-.flex-header-container{
+.flex-header-container {
     background-color: rgb(13, 13, 13);
     margin-top: 0;
     padding: 10px;
     display: flex;
 }
 
-.flex-header-title{
+.flex-header-title {
```

Comment:
recommend to use the full power of scss instead of writing redundant lines of code, so here it would be:

```
.flex-header-title {
    font-size: 24pt;
    font-weight: bold;
    a {
        color: inherit;
        text-decoration: none;
}
```